bundle/language_en.properties

```diff
@@ -1574,6 +1577,18 @@ angal.supplier.requiredfields
 angal.supplier.suppliersbrowser.title                                                                  = Supplier Browser
 angal.supplier.taxcode                                                                                 = Tax Number ID
 angal.supplier.taxcode.col                                                                             = Tax Number ID
+angal.telemetry.about                                                                                  = Collect anonymous usage data and send them to open-hospital.org
```

Comment:
This parameters should have the appropriate suffix, e.g. `title`, `msg`, `txt`,  etc.

Reply:
Fixed, thanks!